src/Audis.Analyzer.Common/Audis.Analyzer.Common/Utils/AdaptiveCardUtils.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using AdaptiveCards.Templating;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Audis.Analyzer.Common.Utils
+{
+    public class AdaptiveCardUtils
+    {
+        public static async Task<string> ConstructAsync(IEnumerable<(string, JObject)> cardTemplates)
```

Comment:
Ich würde hier glaube ich ein `object` statt ein `JObject` verwenden, wie es auch in der [Doku](https://docs.microsoft.com/en-us/adaptive-cards/templating/sdk#usage-1) gemacht wird. Dadurch muss sich der Aufrufer nicht darum kümmern, wie er das JObject erstellt und kann einfach ein anonymes Objekt angeben (`new { wi.Title, ... }`).